src/components/Tabs/Tabs.jsx

```diff
@@ -0,0 +1,127 @@
+import _ from 'lodash';
+import React from 'react';
+import classNames from 'classnames';
+import { bindClassNames } from '../../util/style-helpers';
+import { createLucidComponentDefinition } from '../../util/component-definition';
+import reducers from './Tabs.reducers';
+
+const boundClassNames = bindClassNames('Tabs');
+
+const {
+	any,
+	string,
+	number,
+	bool,
+	func,
+} = React.PropTypes;
+
+/**
+ *
+ * {"categories": ["navigation"]}
+ *
+ * `Tabs` provides tabbed navigation. It has a flexible interface that allows
+ * tab content to be passed as regular React children or through props.
+ */
+const Tabs = React.createClass(createLucidComponentDefinition({
+	displayName: 'Tabs',
+
+	childProps: {
+		Tab: null, // we're only interested in children
+		Title: null, // we're only interested in children
+	},
+
+	reducers,
+
+	propTypes: {
+		/**
+		 * Styles that are passed through to the root container.
+		 */
+		style: any,
+
+		/**
+		 * While you can pass tab content through this prop, it's better to use the
+		 * child component instead. See the examples for details.
+		 */
+		Tab: any.isRequired,
+
+		/**
+		 * `Title` can be a prop on `Tabs.Tab` or a child of `Tabs.Tab`. See the
+		 * examples for details.
+		 */
+		Title: any,
+
+		/**
+		 * Class names that are appended to the defaults.
+		 */
+		className: string,
+
+		/**
+		 * Index of the currently selected tab.
+		 */
+		selectedIndex: number,
+
+		/**
+		 * Callback for when the user clicks a tab. Called with the index of the
+		 * tab that was clicked.
+		 */
+		onSelect: func,
+
+		/**
+		 * If `true` then the active tab will appear open on the bottom.
+		 */
+		isOpen: bool,
+	},
+
+	getDefaultProps() {
+		return {
+			style: null,
+			className: null,
```

Comment:
should we make it a practice to assign `null` in default props? i'm not sure if it's useful